src/dialects/mysql2/index.js

```diff
@@ -26,11 +26,13 @@ assign(Client_MySQL2.prototype, {
   // The "dialect", for reference elsewhere.
   driverName: 'mysql2',
 
+  SqlString: require('../../query/string'),
```

Comment:
This should not be needed since `src/client.js` base class already sets it to `src/query/string.js`